### PR TITLE
Add completion for invoke https://github.com/pyinvoke/invoke

### PR DESCRIPTION
**Pull request by IvanMalison**

### CODE REVIEW

**Comment by wo0dyn** [comment]
Thank you very much @IvanMalison for your work!
:+1: Can't wait to get that plugin merged into oh-my-zsh (/ping @robbyrussell)

File: plugins/invoke/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## git
```
Comment (IvanMalison) [inline plugins/invoke/README.md]: this should read invoke

Comment (ncanceill) [inline plugins/invoke/README.md]: Well, you are the one to modify it.

**Comment by ncanceill** [comment]
Hi, can we get a few more testers here?

**Comment by kenden** [comment]
pyinvoke now has a command to generate a completion script for zsh. I made a new PR to just used it.

**Comment by ThatXliner** [comment]
I made a custom plugin with only one line
```zsh
eval "$(~/.local/bin/invoke --print-completion-script=zsh)"
```

**Comment by kenden** [comment]
@ThatXliner this PR uses the same oneliner
https://github.com/ohmyzsh/ohmyzsh/pull/7311

**Pull request closed without merging**